Fail to upgrade due to quota initialization

I tried to migrate the data when upgrading from 1.8 -> 1.9 
The env has around thousand of images and consume 200+GB of storage. 

While syncing the data for quota usage there's error:

```
Aug 26 10:40:13 172.30.0.1 core[2686]: 2019-08-26T10:40:13Z [ERROR] [/core/api/quota/registry/registry.go:244]: pq: value "4069985900" is out of range for type integer
Aug 26 10:40:13 172.30.0.1 core[2686]: 2019-08-26T10:40:13Z [ERROR] [/core/main.go:102]: Fail to sync quota data, pq: value "4069985900" is out of range for type integer
Aug 26 10:40:13 172.30.0.1 core[2686]: 2019-08-26T10:40:13Z [FATAL] [/core/main.go:242]: quota migration error, pq: value "4069985900" is out of range for type integer
```

--- a/make/migrations/postgresql/0010_1.9.0_schema.up.sql
+++ b/make/migrations/postgresql/0010_1.9.0_schema.up.sql
@@ -16,9 +16,9 @@ CREATE TABLE blob
   /*
      digest of config, layer, manifest
   */
-  digest        varchar(255)       NOT NULL,
-  content_type  varchar(255)       NOT NULL,
-  size          int                NOT NULL,
+  digest        varchar(255)        NOT NULL,
+  content_type  varchar(1024)       NOT NULL,
+  size          bigint              NOT NULL,
   creation_time timestamp default CURRENT_TIMESTAMP,
   UNIQUE (digest)
 );

